src/Components/NavBar/NavBar.tsx

```diff
@@ -3,34 +3,67 @@ import Container from "react-bootstrap/Container";
 import Nav from "react-bootstrap/Nav";
 import Navbar from "react-bootstrap/Navbar";
 import NavDropdown from "react-bootstrap/NavDropdown";
+import { Routes, Route, useNavigate } from "react-router-dom";
+import { StorageKey, useLocalStorage } from "../LocalStorage/LocalStorage";
+import { randomizer } from "../QuestionRandomizer/randomizer";
+
+export const NavBar = ({
+    redirectUserQuestions,
+    redirectUserFlashCard,
+    setShowQuestionList,
+}: any) => {
+    const [userInfo, setUserInfo] = useLocalStorage(StorageKey.userInfo, {
+        email: "",
+        password: "",
+    });
 
-export const NavBar = () => {
     return (
         <div className="NavBar">
-            <Navbar bg="light" expand="lg">
+            <Navbar bg="light" expand="lg" fixed="top">
                 <Container>
                     <Navbar.Brand href="#home">R.T.I.</Navbar.Brand>
                     <Navbar.Toggle aria-controls="basic-navbar-nav" />
                     <Navbar.Collapse id="basic-navbar-nav">
                         <Nav className="me-auto">
-                            <Nav.Link href="#home">Home</Nav.Link>
-                            <Nav.Link href="#link">Question List</Nav.Link>
+                            <Nav.Link href="/flash-card">Home</Nav.Link>
+                            <Nav.Link
+                                onClick={() => {
+                                    setShowQuestionList(true);
+                                    redirectUserQuestions();
+                                }}
+                            >
+                                Question List
+                            </Nav.Link>
+                            <Nav.Link
+                                onClick={() => {
+                                    setShowQuestionList(false);
+                                    redirectUserFlashCard();
```

Comment:
here i put a block of code to update the local storage's current question but would not display the new question when the redirect went to the flash card it would display the old queston. local storage did have a new current question stored though.

should have taken screen shots of the issue.